tests/unit/serialization/custom_global_serialization_test.py

```diff
@@ -128,3 +140,34 @@ def test_double_register_custom_serializer(self):
 
         with self.assertRaises(ValueError):
             service._registry.safe_register_serializer(TheOtherCustomSerializer, CustomClass)
+
+    def test_serializing_class_instances(self):
+        config = Config()
+        config.custom_serializers = {CustomClass: CustomSerializer}
+        service = SerializationServiceV1(config)
+
+        data1 = service.to_data(CustomClass)
+        deserialized1 = service.to_object(data1)
+
+        self.assertEqual(CustomClass, deserialized1)
+
+        data2 = service.to_data(TheOtherCustomClass)
+        deserialized2 = service.to_object(data2)
+
+        self.assertEqual(TheOtherCustomClass, deserialized2)
+
+    def test_serializing_class_and_object_instances(self):
+        config = Config()
+        config.custom_serializers = {CustomClass: CustomSerializer}
+        service = SerializationServiceV1(config)
+
+        data1 = service.to_data(CustomClass)
+        deserialized1 = service.to_object(data1)
+
+        self.assertEqual(CustomClass, deserialized1)
```

Comment:
you already tested that in the above. so, we can remove these three lines

Reply:
I just added to show if there is difference in one test. Removing them